### PR TITLE
Added rotate option

### PR DESCRIPTION
**Pull request by amereii**
Similar to fleaflet/flutter_map#864 I added rotate option to flutter_map_marker_cluster
We must wait for https://github.com/rorystephenson/flutter_map_marker_popup/pull/17 to be merged to remove [dependency_overrides](https://github.com/RooyeKhat-Media/flutter_map_marker_cluster/blob/d81f7d2e26125b0bca6531429691a3b62cd58487/pubspec.yaml#L20-L24)

### CODE REVIEW

**Comment by amereii** [comment]
@lpongetti I wonder how the merging PR might have been done? My changes is not in master branch and the latest release
Can you check it?

File: pubspec.yaml [hunk 1]
```diff
@@ -17,6 +17,12 @@ dependencies:
   latlong: ^0.6.1
   flutter_map_marker_popup: ^0.2.0
 
+dependency_overrides:
```
Comment (lpongetti) [inline pubspec.yaml:20]: why dependency_overrides?

**Comment by amereii** [comment]
@lpongetti As I told

> We must wait for rorystephenson/flutter_map_marker_popup#17 to be merged to remove dependency_overrides

It is not merged as I was too busy to fix it completely, thanks to @rorystephenson now you can remove dependency_overrides and use `flutter_map_marker_popup: ^0.3.0`

File: pubspec.yaml (hunk 1 above)
Comment (amereii) [reply pubspec.yaml:20]: As I told

>We must wait for rorystephenson/flutter_map_marker_popup#17 to be merged to remove dependency_overrides

As I wan too busy to fix it completely it's not merged but thanks to @rorystephenson now you can remove dependency_overrides and use `flutter_map_marker_popup: ^0.3.0`

Comment (lpongetti) [reply pubspec.yaml:20]: pls fix it and reopen

Comment (amereii) [reply pubspec.yaml:20]: @lpongetti I tried to upgrade to `flutter_map_marker_popup: ^0.3.0` but unfortunately `MarkerPopup` was removed and currently I'm too busy to fix it

**Comment by Y0ngg4n** [comment]
Does rotate work now? I don´t find the option for it?

**Comment by rorystephenson** [comment]
> Does rotate work now? I don´t find the option for it?

You should enable rotation in the flutter_map options.

**Comment by Y0ngg4n** [comment]
@rorystephenson I added it to the Marker or what option do you mean?

**Comment by rorystephenson** [comment]
> @rorystephenson i really don´t now how that works: image Rotation is a double in MapOptions. but there is no `rotate` boolean to counter rotate the markers on rotation. I dont know what option do you mean.

This is not the right option. The option for enabling/disabling rotation (as well as various other interaction options) is `interactiveFlags` which should default to 'all' unless you have set it otherwise. Can you confirm whether or not your map is rotatable?

**Pull request merged**